packages/commonwealth/client/scripts/hooks/useInitApp.ts

```diff
@@ -15,6 +15,7 @@ const useInitApp = () => {
           .then((res) => {
             const serverCustomDomain = res.data.customDomain || '';
             setCustomDomain(serverCustomDomain);
+            app.setCustomDomain(serverCustomDomain);
```

Comment:
After a long debug session, this seems to be the problem. Basically the customDomain is not being passed into the Layout.tsx (The `const routerParams = useParams();` resolves to an empty object).